silkshopy-ecommerce/src/pages/ProductListing/functions/sort-price.jsx

```diff
@@ -0,0 +1,12 @@
+const sortByPrice = (sortBy, priceRangeData) => {
+
+    if (sortBy === "LOW_TO_HIGH")
+        return priceRangeData.sort((a, b) => Number(a.price) - Number(b.price));
+
+    if (sortBy === "HIGH_TO_LOW")
+        return priceRangeData.sort((a, b) => Number(b.price) - Number(a.price));
```

Comment:
Here you are using sort function and sort function mutates the original array. So my suggestion is that you can use spread operator while sorting.